tests/integration/io/parquet/test_reads_public_data.py

```diff
@@ -200,6 +200,17 @@ def parquet_file(request) -> tuple[str, str]:
     return request.param
 
 
+@pytest.fixture(params=[(True, True), (True, False), (False, False)], ids=["split", "merge", "ignore"])
+def set_split_config(request):
+    threshold = 0 if request.param[0] else 128 * 1024 * 1024
+    min_size = 0 if request.param[1] else 64 * 1024 * 1024
+
+    daft.set_execution_config(
+        parquet_split_row_groups_threshold_bytes=threshold,
+        parquet_split_row_groups_min_size_bytes=min_size,
+    )
```

Comment:
This execution config will be set for as long as the pytest process lives, i.e. it will be set unintentionally in other tests; we should ensure that this fixture resets the old (default) execution config at the end of each test run to prevent this leakage. This can be done with the good ol' try-yield-finally pattern.

See here for an example of such a fixture!
https://github.com/Eventual-Inc/Daft/blob/6cda37ae7032af8bc0342dc4291cf60be799a69f/tests/dataframe/test_joins.py#L12-L24